docs/resources/User.md

```diff
@@ -89,6 +89,9 @@ Returns the [user](#DOCS_USER/user-object) object of the requester's account. Fo
 
 Returns a [user](#DOCS_USER/user-object) for a given user ID.
 
+>warn
+> Only bot accounts can use the Get User endpoint.
```

Comment:
That warning is also present in `Get Channel Message` which is why I even started this pr to begin with. Should that be removed too?